otter/convergence/steps.py

```diff
@@ -70,12 +102,38 @@ def got_name(random_name):
                 'POST',
                 'servers',
                 data=thaw(server_config),
-                success_pred=has_code(202))
+                success_pred=has_code(202),
+                reauth_codes=(401,))
 
         def report_success(result):
             return StepResult.SUCCESS, []
 
         def report_failure(result):
+            """
```

Comment:
Is that a blocker? If so, why is that function better in a way that won't come out in the (refactoring) wash?